sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -550,6 +552,10 @@ public static BinaryData fromByteBuffer(ByteBuffer data) {
         return new BinaryData(new ByteBufferContent(data));
     }
 
+    public static BinaryData fromList(List<ByteBuffer> data) {
```

Comment:
It would be good to rename this to `fromListByteBuffer` in the case we want to add `List<byte[]>` in the future we can disambiguate the APIs.

Also, will need to add Javadocs for CI to pass.